doc/sphinx/reference/vcl_var.rst

```diff
@@ -510,8 +512,10 @@ req.ttl
 
 	Writable from: client
 
+	Unsetable from: client
 
 	Upper limit on the object age for cache lookups to return hit.
+	Lookups always miss when req.ttl is set to 0s.
```

Comment:
Maybe now would be a good time to document that negative values are swallowed.

Reply:
If. Or document that they are invalid and will trigger a failure.